api/apps/geoprocessing/src/migrations/geoprocessing/1647859641978-AddProjectsPuPlanningAreaIdColumn.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddProjectsPuPlanningAreaIdColumn1647859641978
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      ALTER TABLE projects_pu
+      ADD COLUMN planning_area_id uuid;
+    `);
+    await queryRunner.query(`
+      ALTER TABLE projects_pu
+      ALTER COLUMN project_id DROP NOT NULL;
+    `);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      ALTER TABLE projects_pu
+      DROP COLUMN planning_area_id;
+    `);
+
```

Comment:
At the moment we couldn't rollback this transaction as it would fail with the PUs that have null project_ids. To bypass this error I would add a default fake project uuid like "0000-0000..."